src/components/InventoryTable/InventoryList.js

```diff
@@ -1,72 +1,13 @@
 /* eslint-disable react/display-name */
-import React, { useEffect, useRef } from 'react';
-import { useSelector } from 'react-redux';
+import React from 'react';
 import InventoryEntityTable from './EntityTable';
 import { Grid, GridItem } from '@patternfly/react-core';
 import PropTypes from 'prop-types';
 import './InventoryList.scss';
-import isEqual from 'lodash/isEqual';
 import AccessDenied from '../../Utilities/AccessDenied';
 
-const convertItem = ({ children, isOpen, ...item }) => item;
-
-/**
- * Component that works as a side channel for consumers to notify inventory of new data changes.
- */
-const ContextInventoryList = ({
```

Comment:
This is a redundant wrapper. Removing it will simplify the structure and should reduce the render burden.